IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignment/Documentation.md

```diff
@@ -1,10 +1,63 @@
+For the purposes of IFC the English term "alignment" defines three essentially separate but closely interconnected concepts.
+
+1. defintion of a reference system for linear positioning
+2. safeguarding and optimization of the movment of vehicles - kinematic perspective
+3. geometric construction of roads, railway tracks or other linear infrastucture
+
+**Reference system for linear positioning**
+
 An alignment is used to define a reference system to position elements mainly for linear construction works, such as roads, rails, bridges, and others. The relative positioning along the alignment is defined by the linear referencing methodology.  
   
 > NOTE&nbsp; See ISO 19148 Geographic information &ndash; Linear referencing for general definitions about linear referencing.  
+
+
+**Kinematic perspective**
+
+In the kinematic perspective focus is on the safe and optimized movement of a vehicle under the constraints induced by changes in the direction of the horizontal and the vertical layout.
+
+
+**Geometric perspective**
+
+In the geometric perspective the focus is on the proper placement of horizontal and vertical segments to connect certain points along a proposed path. A huge body of knowledge has been developed over a long period of time, in many aspects predating the availability of modern computers and their software.
+
+
+
+**State of the art in contemporary engineering**
+
+1. Contemporary engineering usually establishes first a horizontal layout in a properly projected plane.
+
+2. In a second step the vertical profile (i.e. sequence of segments with constant gradients) is added.
```

Comment:
Is it only a sequence of constant gradients? When are the ones with changing gradients added?

Reply:
Well spotted. Will modify.